pkg/config/config.go

```diff
@@ -1158,7 +1158,7 @@ func ExtractConfigData(yamlData []byte) (common.Configuration, error) {
 
 	if segutils.ConvertUintBytesToMB(totalMemory) < SIZE_8GB_IN_MB {
 		if memoryLimits.MaxUsagePercent > 50 {
-			log.Infof("ExtractConfigData: MemoryThresholdPercent is set to %v%% but bringing it down to 50%%", memoryLimits.MaxUsagePercent)
+			log.Infof("ExtractConfigData: Max Memory Usage Percent is set to %v%% but bringing it down to 50%%", memoryLimits.MaxUsagePercent)
```

Comment:
Use the exact name in the log line as it is defined in config so that it becomes easier for user to debug